streams/src/test/java/org/apache/kafka/test/MockRecordCollector.java

```diff
@@ -46,13 +47,15 @@ public <K, V> void send(final String topic,
                             final Integer partition,
                             final Long timestamp,
                             final Serializer<K> keySerializer,
-                            final Serializer<V> valueSerializer) {
+                            final Serializer<V> valueSerializer,
+                            final String processorNodeId,
+                            final InternalProcessorContext<Void, Void> context) {
         collected.add(new ProducerRecord<>(topic,
-            partition,
-            timestamp,
-            key,
-            value,
-            headers));
+                                           partition,
+                                           timestamp,
+                                           key,
+                                           value,
+                                           headers));
```

Comment:
nit: The indentation was actually right since it is a method call and not a method declaration.

Reply:
Ah my IDE probably autoformatted it - will fix